src/theme/_content.scss

```
@@ -23,6 +23,18 @@
       letter-spacing: 1px;
       text-transform: uppercase;
     }
+
+    @container (min-width: #{$largest-mobile-screen} ) {
+      .day + .headtitle::before {
```

```suggestion
      .day + .head-title::before {
```
(I renamed the class in another PR to match some existing styles.)